backend/main.go

```diff
@@ -520,6 +520,8 @@ func main() {
 			r.Use(public.PublicMiddleware)
 			r.Use(highlightChi.Middleware)
 
+			r.Post("/login", publicResolver.Login)
```

Comment:
could we use the `PrivateGraph` instead for login? the 'public graph' is meant for our unauthenticated data ingest, while the 'private graph' is the api for retrieving data for our frontend.

the `/login` route itself should be unauthenticated on the private graph, since it is what will establish the auth token (it should be set up before the private graph middleware that makes private graph require authentication).

Reply:
use PrivateGraph for "login"